### ISSUE TITLE
Tracee Helm Installation Fails: missing kernel symbol dependency

### ISSUE THREAD

**Issue by jicunningham** [aquasecurity/tracee#3362]
## Description

Unable to install Tracee via helm repo in GCP. Command run:

`helm install tracee aqua/tracee \
        --namespace tracee-system --create-namespace \
        --set hostPID=true`

When checking the logs of the pods, I can see this message:

`Found 5 pods, using pod/tracee-5544l
{"level":"error","ts":1691077966.0057297,"msg":"Event canceled because of missing kernel symbol dependency","missing symbols":["sys_call_table"],"event":"print_syscall_table"}
{"level":"warn","ts":1691077991.8121207,"msg":"libbpf: prog 'trace_vfs_writev': failed to create kprobe 'vfs_writev+0x0' perf event: No such file or directory"}
{"level":"warn","ts":1691077991.829119,"msg":"libbpf: prog 'trace_ret_vfs_writev': failed to create kretprobe 'vfs_writev+0x0' perf event: No such file or directory"}
TIME             UID    COMM             PID     TID     RET              EVENT                     ARGS`

I am not sure why this is happening. I also pulled the helm chart locally and deployed that way but ran into the same issue.

## Additional details 

I also received this error:

`Readiness probe failed: Get "http://myIP:3366/healthz": dial tcp myIP:3366: connect: connection refused`

**Comment by jicunningham** [comment]
What is also interesting is that the individual tracee pods will pick up logs and report back if the container in question is on the same node as the pod with the shady behavior but I won't get continuous logs from the deployment like shown in the tutorials via helm. Here is the logs from a tracee pod on the same node as a dummy ubuntu pod that I tried to trigger it:

```
> kubectl logs tracee-5cvhm -n tracee-system
{"level":"error","ts":1691178128.0150187,"msg":"Event canceled because of missing kernel symbol dependency","missing symbols":["sys_call_table"],"event":"print_syscall_table"}
{"level":"warn","ts":1691178136.7610817,"msg":"libbpf: prog 'trace_vfs_writev': failed to create kprobe 'vfs_writev+0x0' perf event: No such file or directory"}
{"level":"warn","ts":1691178136.775817,"msg":"libbpf: prog 'trace_ret_vfs_writev': failed to create kretprobe 'vfs_writev+0x0' perf event: No such file or directory"}
TIME             UID    COMM             PID     TID     RET              EVENT                     ARGS
00:50:46:785378  0      sysctl           2324692 2324692 0                aslr_inspection           
06:50:46:663752  0      sysctl           2391484 2391484 0                aslr_inspection           
12:50:46:683316  0      sysctl           2458345 2458345 0                aslr_inspection           
18:50:46:619910  0      sysctl           2525237 2525237 0                aslr_inspection           
00:50:46:722980  0      sysctl           2591774 2591774 0                aslr_inspection           
06:50:46:653741  0      sysctl           2658080 2658080 0                aslr_inspection           
12:50:46:688574  0      sysctl           2725122 2725122 0                aslr_inspection           
18:50:46:673810  0      sysctl           2792846 2792846 0                aslr_inspection           
00:50:46:699323  0      sysctl           2860714 2860714 0                aslr_inspection           
06:50:46:664570  0      sysctl           2928384 2928384 0                aslr_inspection          
```


Below are the logs I am seeing after I deleted the ubuntu pod and it got assigned to a new node. These are logs from the tracee pod that is on the same node:

```
{"level":"error","ts":1691287215.0058243,"msg":"error processing control plane signal","error":"controlplane.(*Controller).processCgroupMkdir: failed to get lookup key 824648583784 from map containers_map: no such file or directory"}
{"level":"error","ts":1691290335.7848806,"msg":"error processing control plane signal","error":"controlplane.(*Controller).processCgroupMkdir: failed to get lookup key 824663770840 from map containers_map: no such file or directory"}
{"level":"error","ts":1691301138.1921563,"msg":"error processing control plane signal","error":"controlplane.(*Controller).processCgroupMkdir: failed to get lookup key 824649751712 from map containers_map: no such file or directory"}
07:49:41:536520  0      sysctl           2742830 2742830 0                aslr_inspection           
{"level":"error","ts":1691312420.5420518,"msg":"error processing control plane signal","error":"controlplane.(*Controller).processCgroupMkdir: failed to get lookup key 824655407368 from map containers_map: no such file or directory"}
{"level":"error","ts":1691322742.80687,"msg":"error processing control plane signal","error":"controlplane.(*Controller).processCgroupMkdir: failed to get lookup key 824675929264 from map containers_map: no such file or directory"}
{"level":"error","ts":1691326343.6515234,"msg":"error processing control plane signal","error":"controlplane.(*Controller).processCgroupMkdir: failed to get lookup key 824676182240 from map containers_map: no such file or directory"}
13:49:41:549291  0      sysctl           2810204 2810204 0                aslr_inspection           
{"level":"error","ts":1691337621.3761024,"msg":"error processing control plane signal","error":"controlplane.(*Controller).processCgroupMkdir: failed to get lookup key 824654788672 from map containers_map: no such file or directory"}
{"level":"error","ts":1691340746.767978,"msg":"error processing control plane signal","error":"controlplane.(*Controller).processCgroupMkdir: failed to get lookup key 824654534312 from map containers_map: no such file or directory"}
{"level":"error","ts":1691341226.81246,"msg":"error processing control plane signal","error":"controlplane.(*Controller).processCgroupMkdir: failed to get lookup key 824655224376 from map containers_map: no such file or directory"}
19:49:41:576293  0      sysctl           2877676 2877676 0                aslr_inspection           
{"level":"error","ts":1691352018.8127654,"msg":"error processing control plane signal","error":"controlplane.(*Controller).processCgroupMkdir: failed to get lookup key 824659611560 from map containers_map: no such file or directory"}
{"level":"error","ts":1691355149.9992056,"msg":"error processing control plane signal","error":"controlplane.(*Controller).processCgroupMkdir: failed to get lookup key 824647773584 from map containers_map: no such file or directory"}
{"level":"error","ts":1691362351.5826259,"msg":"error processing control plane signal","error":"controlplane.(*Controller).processCgroupMkdir: failed to get lookup key 824663774776 from map containers_map: no such file or directory"}
01:49:41:640116  0      sysctl           2944197 2944197 0                aslr_inspection           
{"level":"error","ts":1691387557.137988,"msg":"error processing control plane signal","error":"controlplane.(*Controller).processCgroupMkdir: failed to get lookup key 824676638576 from map containers_map: no such file or directory"}
{"level":"error","ts":1691388009.852133,"msg":"error processing control plane signal","error":"controlplane.(*Controller).processCgroupMkdir: failed to get lookup key 824659337336 from map containers_map: no such file or directory"}
07:49:41:604092  0      sysctl           3011368 3011368 0                aslr_inspection           
{"level":"error","ts":1691409638.4021583,"msg":"error processing control plane signal","error":"controlplane.(*Controller).processCgroupMkdir: failed to get lookup key 824679155688 from map containers_map: no such file or directory"}
12:30:15:416563  0      dpkg             19258   19258   14488            dropped_executable        path: /usr/bin/locale-check.dpkg-new
12:44:53:277942  0      strace           19717   19717   0                anti_debugging
```

Though it has what I am looking for in terms of abnormal behavior (I ran an `apt update && apt upgrade -y`) the timestamps are inconsistent. The times starting with `12:30` and `12:44` are from the commands I just ran previous to updating this issue. I am assuming they are from previous days but maybe a feature request would be to include the date in those timestamps as well?

**Comment by josedonizetti** [comment]
Since the helm installation was fixed on the release 0.17.1, I will close this issue, though I've extract another two issues from here which will be investigating separately:

- https://github.com/aquasecurity/tracee/issues/3397
- https://github.com/aquasecurity/tracee/issues/3396
